src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+
+    add(num) {
+      calc.result += num;
+    },
+
+    subtract(num) {
+      calc.result -= num;
+    },
+
+    multiply(num) {
+      calc.result *= num;
+    },
+
+    divide(num) {
+      calc.result /= num;
+    },
+
+    operate(callback, num) {
+      callback(num);
+
+      return calc;
```

Comment:
In the `operate` method, consider using `this` instead of `calc` to maintain the context of the current calculator instance. This is more aligned with standard practices and helps if the method is ever used in a different context than the original object.